books/RayTracingInOneWeekend.html

```diff
@@ -1391,19 +1391,20 @@
 We need some math constants that we conveniently define in their own header file. For now we only
 need infinity, but we will also throw our own definition of pi in there, which we will need later.
 There is no standard portable definition of pi, so we just define our own constant for it. We'll
-throw common useful constants and future utility functions in `rtweekend.h`, our general main header
-file.
+also throw common useful constants and future utility functions in `rtweekend.h`, our general main
+header file.
 
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
     #ifndef RTWEEKEND_H
     #define RTWEEKEND_H
 
     #include <cmath>
+    #include <iostream>
```

Comment:
Should there be a `#include <cstdlib>` here to match the code?

Reply:
Will fix.

Reply:
Nope, not needed yet.

Reply:
See line 2002.

Reply:
Ah, yeah I missed its introduction down the line.